testing/test-infra/app.yaml

```diff
@@ -1,12 +1,6 @@
 apiVersion: 0.0.1
 kind: ksonnet.io/app
 libraries:
-  core:
```

Comment:
Keep this and check in vendor.  We want to check in vendor so that the code in the repo is sufficient. Please update PR description accordingly.

Reply:
Done